scripts/make-request.py

```diff
@@ -0,0 +1,19 @@
+import requests, os, json
+import logging
+try:
+    from http.client import HTTPConnection # py3
+except ImportError:
+    from httplib import HTTPConnection # py2
+
+# enable verbose logging
```

Comment:
need to ensure we're not logging the api key

Reply:
does not seem to log to worker console https://github.com/MetaMask/dns/actions/runs/6671651594/job/18135339792

Reply:
api key is logged but censored by github

Reply:
disabled